server/routes/appointments/appointmentSummaryComponent.ts

```diff
@@ -0,0 +1,72 @@
+import { SummaryListItem } from '../../utils/summaryList'
+import { ActionPlanAppointment } from '../../models/actionPlan'
+import Appointment from '../../models/appointment'
+import AppointmentDecorator from '../../decorators/appointmentDecorator'
+import PresenterUtils from '../../utils/presenterUtils'
+import AuthUserDetails from '../../models/hmppsAuth/authUserDetails'
+
+export default class AppointmentSummaryComponent {
+  constructor(
+    private readonly appointment: Appointment | ActionPlanAppointment,
+    private readonly assignedCaseworker: AuthUserDetails | null = null
+  ) {}
+
+  private readonly appointmentDecorator = new AppointmentDecorator(this.appointment)
+
+  get appointmentSummary(): SummaryListItem[] {
```

Comment:
maybe we can call this `.summaryList` if we rename the class as suggested above.

Reply:
I've renamed to appointmentSummaryList